server/models/Conversation.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const conversationSchema = new Schema(
+  {
+    user1: {
+      type: Schema.Types.ObjectId,
+      ref: "user",
+    },
+    user2: {
+      type: Schema.Types.ObjectId,
+      ref: "user",
+    },
+    messages: [
```

Comment:
Optional: As opposed to storing all the messages within a conversation, maybe consider storing only the lastMessage within a conversation? :)

Reply:
Applied. 🙂

I understand why you suggest storing the `latestMessage` in a `Conversation` model regarding `GET /conversations`.
So, I modified `Conversation` and `Message` models. (Replaced `messages[]` with `latestMessage`)
Also, I modified some logic and `seed.js` as well. It seems to work fine. 

1. But, I am still a bit confused. To get all messages in a conversation, isn't it better to keep the previous version(`messages[]`)? 🤔

2. With these changes, is it correct to use `Message.find(...)` in `GET /conversations/:conversationId` to get all messages in a conversation?

Reply:
Hey @jstyle5, please see below:

1.  The previous version works perfectly fine as well but we may have longer response times while fetching all the conversations. Take for an example, we have 50 conversations with 1000 messages each, each time we fetch those conversations, we are fetching 50000 messages with them 😄 

2. Yes, it is correct to use `Message.find(...)` with the changes we have made.

Reply:
> Hey @jstyle5, please see below:
> 
> 1. The previous version works perfectly fine as well but we may have longer response times while fetching all the conversations. Take for an example, we have 50 conversations with 1000 messages each, each time we fetch those conversations, we are fetching 50000 messages (or message ids) with them 😄
> 2. Yes, it is correct to use `Message.find(...)` with the changes we have made.

Ah, I see. That's true. Thanks for your feedback. 😄